ui/apiserver/apiserver.go

```diff
@@ -25,10 +25,11 @@ const (
 	redisQueueArchive  = "archived"
 )
 
-const PreCheckEndpointURL = "https://merlinite-7b-vllm-openai.apps.fmaas-backend.fmaas.res.ibm.com/v1"
+const PreCheckEndpointURL = "http://cmb-staging.asgharlabs.io:8505/v1"
```

Comment:
I can revert this piece. In anycase we do have the ability to set this in the env with the `PECHECK_ENDPOINT` var which would make this a runtime rather than code config.